### DISCUSSION TITLE
Exit dialog box buttons' texts are misleading*

### DISCUSSION THREAD

**Discussion by Paladynee** [ppy/osu#23981] [Ideas] [answered]
As a long time computer user, whenever I hit the X button to leave the program, I'm either prompted with a clear Exit or Cancel button, and I instinctively know what to click on, without needing to read/parse the meanings of Exit or Cancel individually.

However on osu!lazer, whenever I try to exit, it prompts me with "Yes, let me out!" and "No, I would like to go back" or whatever (i dont remember). I'm frustrated because I have to read and parse the meanings of the buttons in order to decide what to click. Out of habit, this sometimes makes me not read what the buttons say, and
### *: **I accidentally click the Cancel button**.

Both the accidental clicks and the reading the text build up over time and add to the frustration.

This dialogue box that I'm speaking of appears whenever I update lazer and click on the update notification to restart my game. Whenever I manually click the Exit button on the main menu, I'm not shown any dialogue boxes.

### A very cool QoL fix for this would be either 1: falling back to Exit/Cancel or 2: letting the player edit box texts.

**Comment by peppy** [comment]
I agree the text is maybe too long, but do the colours of the buttons not help in this scenario?

**Reply by Paladynee** [reply]
Both of them being very similar in brightness and ~~used in vast amounts throughout the app~~ matching the app theme in general does not help, maybe making the exit button more saturated would help.

**Reply by peppy** [reply, marked as answer]
I've made some progress towards making the scenario you mention better (see #23987)
